src/Concordium/Client/Runner.hs

```diff
@@ -170,10 +170,10 @@ getFromJsonAndHandleError handleError r = do
 
 -- |Look up account from the provided name or address. If 'Nothing' is given, use the defaultAcccountName.
 -- Fail if the address cannot be found.
--- Should be kept consistent with processAccountCmd for AccountShow
 getAccountAddressArg :: AccountNameMap -> Maybe Text -> IO NamedAddress
-getAccountAddressArg m input =
-  case getAccountAddress m $ fromMaybe defaultAccountName input of
+getAccountAddressArg m input = do
```

Comment:
As far as I can tell this function is only ever used with the second argument being `Just _`. So it can be simplified.